src/linux/system.rs

```diff
@@ -756,12 +756,11 @@ fn parse_stat_file(data: &str) -> Result<Vec<&str>, ()> {
     let mut parts = Vec::with_capacity(52);
     let mut data_it = data.splitn(2, ' ');
     parts.push(unwrap_or_return!(data_it.next()));
-    // The following loses the ) from the input, but that's ok because
-    // we're not using it anyway.
     let mut data_it = unwrap_or_return!(data_it.next()).rsplitn(2, ')');
     let data = unwrap_or_return!(data_it.next());
     parts.push(unwrap_or_return!(data_it.next()));
     parts.extend(data.split_whitespace());
+    parts[1] = &parts[1][1..]; // Remove command name '('
```

Comment:
What about something like this:

```suggestion
    if let Some(part) = parts.get_mut(1) {
        if part.starts_with('(') {
            part = &part[1..];
        }
    }
```

?

Reply:
I have moved to `strip_prefix` if that is ok.